.eslintrc.cjs

```diff
@@ -1,15 +1,20 @@
-"use strict";
```

Comment:
This module is still CJS.

Content of `.eslintrc.cjs` should be:

```js
"use strict";

module.exports = {
    root: true,
    extends: "eslint",
    env: {
        es6: true
    },
    overrides: [
        {
            files: ["tests/lib/**"],
            parserOptions: {
                ecmaVersion: 2020
            },
            env: {
                mocha: true
            }
        }
    ]
};

```

[`eslint-plugin-node:recommended`](https://github.com/mysticatea/eslint-plugin-node/blob/master/lib/configs/recommended.js) (which we extend from `eslint-config-eslint`) already applies different configurations for `.js` and `.cjs` files.